yalla-coop/tempo
# 590 update member sign up

#### Issues this PR relates to
#590 
how to test : 
  - get the invitation link from earn group account 
  - try to sign up as a member using the invitation link and you will see the updates in the sign up page 
  - after sign up successfully it will direct you to the continue sign up page to complete the sign and then will direct you to Welcom page 
  - if you clicked skip this page will direct you directly to Welcome page 
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs
